HTML_CSS/README.md

```diff
@@ -0,0 +1,61 @@
+## HTML/CSS challenge assignments for Wealth42
+
+### Introduction
+The goal of this challenge is to test your familiarity with concepts in HTML & CSS.
+To read more about the recruitment process and open opportunities at Wealth42, click [here](http://bit.ly/w42-hiring)
+
+### Pick an evaluation mode
+
+1.   **Project of your choice**: You are free to submit a project of your own with *some* conditions.
+     *   Qualifying criteria for submission:
```

Comment:
Lets do the following changes:
- remove qualifying and additional criteria
- move all criteria points as direct bullet points of 'project of your choice'